package.json

```diff
@@ -71,12 +76,21 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.16.0",
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/api-logs": "^0.206.0",
+    "@opentelemetry/core": "^2.1.0",
+    "@opentelemetry/resources": "^2.1.0",
+    "@opentelemetry/sdk-logs": "^0.206.0",
+    "@opentelemetry/sdk-trace-base": "^2.1.0",
+    "@opentelemetry/sdk-trace-node": "^2.1.0",
+    "@opentelemetry/semantic-conventions": "^1.37.0",
     "@types/body-parser": "^1.19.6",
     "@types/bun": "latest",
     "@types/connect": "^3.4.38",
     "@types/node": "^22.10.1",
     "@typescript-eslint/eslint-plugin": "^8.18.0",
     "@typescript-eslint/parser": "^8.18.0",
+    "@vercel/otel": "^2.0.1",
```

Comment:
These new dependencies should be under "dependencies", because we depend on them at runtime, correct?

Reply:
Tidewave relies on some otel objects/classes too, no? I just looked at `dist/next-js/logger.js` and it looks like we bundle some parts of otel actually. We should rather have all of those as external in the bundling process and the necessary package should be actual dependencies, so they are installed together with tidewave. Otherwise, the user may install otel, but tidewave comes with bundled different implementation of otel and it may break (?)

Reply:
Yeah, correct. At least tidewave is a dev dependency, so it should be more manageable.

If we wanted to separate, I think we would need to publish separate packages: `@tidewave/js` (cli), `@tidewave/vite` and `@tidewave/next`. This may be a more idiomatic approach, but I am not sure if it is a good idea to start with multiple packages. Perhaps we can migrate to that structure once people try it out and it gets more stable. cc @josevalim

Reply:
sure, i don't think we should split rn